pkg/master/master.go

```diff
@@ -570,16 +570,30 @@ func (m *Master) init(c *Config) {
 	requestInfoResolver := &apiserver.APIRequestInfoResolver{APIPrefixes: sets.NewString(strings.TrimPrefix(defaultVersion.Root, "/")), RestMapper: defaultVersion.Mapper}
 	apiserver.InstallServiceErrorHandler(m.handlerContainer, requestInfoResolver, apiVersions)
 
+	// allAPIVersions records all AVIVersions that are supported at /apis, in the
+	// form of "group/version"
+	allAPIVersions := []string{}
+
 	if m.exp {
 		expVersion := m.experimental(c)
 		if err := expVersion.InstallREST(m.handlerContainer); err != nil {
 			glog.Fatalf("Unable to setup experimental api: %v", err)
 		}
-		apiserver.AddApiWebService(m.handlerContainer, c.ExpAPIPrefix, []string{expVersion.Version})
+		g, err := latest.Group("experimental")
```

Comment:
I'd like to see a loop here that goes through every Group, instead of specifically referencing experimental. But we can fix this when we add additional groups, I guess.